contracts/Strategy.sol

```diff
@@ -637,18 +634,22 @@ contract Strategy is BaseStrategy, DydxFlashloanBase, ICallee {
     //lets leave
     //if we can't deleverage in one go set collateralFactor to 0 and call harvest multiple times until delevered
     function prepareMigration(address _newStrategy) internal override {
-        (uint256 deposits, uint256 borrows) = getLivePosition();
-        _withdrawSome(deposits.sub(borrows), false);
 
-        (, , uint256 borrowBalance, ) = cToken.getAccountSnapshot(address(this));
+        if(!forceMigrate){
```

Comment:
Can you expand here on what this flag does or add some docs? the name is a bit confusing, it seems by default this being FALSE we try to go thru with the migration, so what are we forcing when this flag is true?